File: search_rebranding.js
const fs = require('fs');
const path = require('path');

const searchTerms = ["lumiere", "lumière"];

function searchFiles(dir) {
    const files = fs.readdirSync(dir);
    
    files.forEach(file => {
        const fullPath = path.join(dir, file);
        const stats = fs.statSync(fullPath);
        
        if (stats.isDirectory()) {
            if (file === 'node_modules' || file === '.git' || file === 'dist') return;
            if (searchTerms.some(term => file.toLowerCase().includes(term))) {
                console.log(`FOLDER MATCH: ${fullPath}`);
            }
            searchFiles(fullPath);
        } else {
            if (searchTerms.some(term => file.toLowerCase().includes(term))) {
                console.log(`FILE NAME MATCH: ${fullPath}`);
            }
            
            try {
                const content = fs.readFileSync(fullPath, 'utf8');
                if (searchTerms.some(term => content.toLowerCase().includes(term))) {
                    console.log(`CONTENT MATCH: ${fullPath}`);
                }
            } catch (e) {}
        }
    });
}

searchFiles('.');

File: scripts/legacy_utils/search_rebranding.py
import os

def search_lumiere():
    search_terms = ["lumiere", "lumière"]
    for root, dirs, files in os.walk("."):
        if "node_modules" in dirs:
            dirs.remove("node_modules")
        if ".git" in dirs:
            dirs.remove(".git")
        if "dist" in dirs:
            dirs.remove("dist")
            
        for name in dirs:
            if any(term in name.lower() for term in search_terms):
                print(f"FOLDER MATCH: {os.path.join(root, name)}")
        
        for name in files:
            if any(term in name.lower() for term in search_terms):
                print(f"FILE NAME MATCH: {os.path.join(root, name)}")
            
            filepath = os.path.join(root, name)
            try:
                with open(filepath, 'r', encoding='utf-8') as f:
                    content = f.read()
                    if any(term in content.lower() for term in search_terms):
                        print(f"CONTENT MATCH: {filepath}")
            except:
                pass

if __name__ == "__main__":
    search_lumiere()
